src/components/Activities/Activity2/index.tsx

```diff
@@ -1,244 +1,77 @@
-import React, { useContext, useState } from 'react';
+import React, { useState } from 'react';
 
+import Clock from '../../../assets/clock.svg';
+import Carousel from '../../shared/Carousel';
 import '../../styles/Activity1.scss';
 import '../../styles/Activity2.scss';
 import '../../styles/CompressionGame.scss';
 
-import AlienFlowerGif from '../../../assets/activity2/game/alien-flower.gif';
-import AlienWigGif from '../../../assets/activity2/game/alien-wig.gif';
-import AlienGif from '../../../assets/activity2/game/alien.gif';
-
-import ElephantHatGIF from '../../../assets/activity2/game/elephant-hat.gif';
-import ElephantShoesGIF from '../../../assets/activity2/game/elephant-shoes.gif';
-import ElephantGIF from '../../../assets/activity2/game/elephant.gif';
-
-import * as A2Animation1 from '../../../assets/lottie_animation_data/activity2/1VideosLotsInfo.json';
-import * as A2Animation2 from '../../../assets/lottie_animation_data/activity2/2TrySendWholeVideo.json';
-import * as A2Animation3 from '../../../assets/lottie_animation_data/activity2/3VerySlow.json';
-import * as A2Animation4 from '../../../assets/lottie_animation_data/activity2/4HaveToLightenLoad.json';
-import * as A2Animation5 from '../../../assets/lottie_animation_data/activity2/5CompressVideo.json';
-import * as A2Animation6 from '../../../assets/lottie_animation_data/activity2/6RocketLighter.json';
-import * as A2Animation7 from '../../../assets/lottie_animation_data/activity2/7LikeInstructions.json';
-import * as A2Animation8 from '../../../assets/lottie_animation_data/activity2/8FinalVideo.json';
-
-import Carousel, { CarouselContext } from '../../shared/Carousel';
-import GIFPlayer from '../../shared/GIFPlayer';
-import LottieControl from '../../shared/LottieControl';
-import { AnswerDisplayStyles } from '../../shared/PlaynetConstants';
 import CompressionGame from './Game';
+import compressedSlides from './Game/compressedSlides';
+import Intro from './Game/Intro';
+import Outro from './Game/Outro';
+import uncompressedSlides from './Game/uncompressedSlides';
+import IntroSlides from './IntroSlides';
 
 function Activity2(): JSX.Element {
-  const [gameTimes, setGameTimes] = useState<string[]>(['-1', '-1', '-1', '-1', '-1', '-1']);
-  const storage = window.sessionStorage;
-
-  const setTimeElapsed = (gameNum: number, slideNum: number, time: number) => {
-    const index = gameNum * 3 + slideNum;
-    time = time / 1000; //convert milliseconds to seconds
-    storage.setItem('A2MinigameIndex' + String(index), String(time));
+  const [uncompressedTime, setUncompressedTime] = useState(0);
+  const [compressedTime, setCompressedTime] = useState(0);
+  const setUncompressedTimeElapsed = (time: number) => setUncompressedTime(uncompressedTime + time);
```

Comment:
and also for some reason its not saving?? idk why tho, but ig sessionStorage would also fix :)
```suggestion
  const setUncompressedTimeElapsed = (time: number) => {console.log(time + " " + uncompressedTime); setUncompressedTime(uncompressedTime + time)};
```
shows its not saving

Reply:
hmmm... act 2 is rerenders like crazy :(

I think we will have to make it a reference

Reply:
wait mine is saving?

Reply:
i was talking to darren and his doesnt save either for some reason :0, but i also dont know why? i also added a useeffect and it seems like its saving at least once

Reply:
ah i see the problem, so actually, its the function

what the function is doing is taking the initial value of uncompressed time and adding that to time..

which means that it will always add zero with time

Reply:
oooooooooo interesting icic